### PR TITLE
Adding assignment

### PR DESCRIPTION
**Pull request by PetJs**

### CODE REVIEW

File: Fagoroye Peter/composition.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        this.powerState = true;
+        return "Turning on the Laptop...."
+    }
+
+    switchOff(){
+        this.powerState = false;
+        return "Turning off the Laptop...."
+    }
+
+    update(){
```
Comment (tdadadavid) [inline Fagoroye Peter/composition.ts:39]: This is not the assingment.
Meet 'Elochuwkwu' for explanation

**Review by tdadadavid — COMMENTED** [review]
Where is your shopping cart assignment?

File: Fagoroye Peter/composition.ts (hunk 1 above)
Comment (PetJs) [reply Fagoroye Peter/composition.ts:39]: when i push the commit, we haven't been assigned the shopping cart yet, I would do that now

Comment (PetJs) [reply Fagoroye Peter/composition.ts:39]: can i still add the shopping cart since you have closed the pull request
or i would have to open another one sir

**Pull request closed without merging**